mithril-common/src/messages/certificate_pending.rs

```diff
@@ -0,0 +1,88 @@
+use serde::{Deserialize, Serialize};
+
+use crate::entities::{Beacon, ProtocolParameters, Signer};
+
+/// Structure to transport [crate::entities::CertificatePending] data.
+#[derive(Clone, Debug, PartialEq, Default, Serialize, Deserialize)]
+pub struct CertificatePendingMessage {
+    /// Current Beacon
+    pub beacon: Beacon,
+
+    /// Current Protocol parameters
+    #[serde(rename = "protocol")]
+    pub protocol_parameters: ProtocolParameters,
+
+    /// Next Protocol parameters
+    #[serde(rename = "next_protocol")]
+    pub next_protocol_parameters: ProtocolParameters,
+
+    /// Current Signers
+    pub signers: Vec<Signer>,
+
+    /// Signers that will be able to sign on the next epoch
+    pub next_signers: Vec<Signer>,
+}
+
+impl CertificatePendingMessage {
+    /// Provide a dummy instance for test.
+    pub fn dummy() -> Self {
+        Self {
+            beacon: Beacon::default(),
+            protocol_parameters: ProtocolParameters {
+                k: 5,
+                m: 100,
+                phi_f: 0.65,
+            },
+            next_protocol_parameters: ProtocolParameters {
+                k: 5,
+                m: 100,
+                phi_f: 0.65,
+            },
```

Comment:
Maybe a different value for the next protocol parameters?
```suggestion
            next_protocol_parameters: ProtocolParameters {
                k: 50,
                m: 1000,
                phi_f: 0.65,
            },
```